warehouse/models/gtfs_schedule_latest_only/_gtfs_schedule_latest_only.yml

```diff
@@ -8,16 +8,20 @@ models:
       Each row is a cleaned row from an agency.txt file.
       Definitions for the original GTFS fields are available at:
       https://gtfs.org/reference/static#agencytxt.
+    meta:
+      ckan.authority: https://developers.google.com/transit/gtfs/reference#agencytxt
```

Comment:
These should reference gtfs.org not the Google Developers site

Reply:
Ah yeah weird, tbh I did not realize they were different in our current metadata.